packages/wrangler/src/logger.ts

```diff
@@ -31,17 +31,6 @@ const getLogLevelFromEnv = getEnvironmentVariableFactory({
 	variableName: "WRANGLER_LOG",
 });
 
-const warnOnce = (() => {
-	let logged = false;
-
-	return (...args: Parameters<typeof console.warn>) => {
-		if (!logged) {
-			console.warn(...args);
```

Comment:
This previously used console.warn instead of logger.warn

This helper has been replaced by logger.warnOnce which will use logger.warn